grpc.go

```diff
@@ -688,7 +668,7 @@ func (a *agentGRPC) CreateSandbox(ctx context.Context, req *pb.CreateSandboxRequ
 		}
 	}
 
-	mountList, err := addMounts(req.Storages)
+	mountList, err := addStorages(req.Storages)
```

Comment:
So the rootfs is passed as part of Storage. But one thing not clear to me is how we indicate the rootfs to libcontainer.  I remember OCI config has explicit field called "Rootfs". Looks like we are not populating that .

Reply:
The OCI spec has a field `ociSpec.Root.Path` specifying the rootfs. But in this case, we don't need to make the diff between a rootfs and a generic storage since for both we expect to mount them somewhere. Different handlers (`blk`, `scsi`, `9p`) will act differently.

Reply:
@sboeuf I assume the rootfs is mounted from the Storage[] depending on its location i.e it will be the first to be mounted. I do not particularly like this as this makes the code unclear and prone to errors. I would have wanted to see an explicit field that indicates the rootfs for a container. The last I saw libcontainer does a few setup steps for setting up rootfs. I am wondering if we are not using the libcontainer api the way it is meant to be used.